util/topsql/topsql.go

```diff
@@ -50,16 +50,18 @@ func SetupTopSQL() {
 	singleTargetDataSink = reporter.NewSingleTargetDataSink(remoteReporter)
 
 	globalTopSQLReport = remoteReporter
-
-	tracecpu.GlobalSQLCPUProfiler.SetCollector(remoteReporter)
-	tracecpu.GlobalSQLCPUProfiler.Run()
 	stmtstats.SetupAggregator()
 }
 
+// SetupTopSQLForTest sets up the global top-sql reporter, it's exporting for test.
+func SetupTopSQLForTest(r reporter.TopSQLReporter) {
+	globalTopSQLReport = r
+}
+
```

Comment:
Maybe you will find your unit testing easier if it is not a "global", but a "default". The `package.DefaultXxx` pattern is widely used to make the API easy to use while maintaining testability, as for something `Default`, there is still possibility to be not a default, which makes it fully testable.